mobile/src/home-view.js

```diff
@@ -126,6 +125,14 @@ export default class HomeView extends PureComponent {
       </View>
     )
   }
+
+  returnIsDisplayable = (s, sessions) => {
```

Comment:
It's not idiomatic to call a function `returnXyz`.  Here, `isDisplayable` is a fine name for the function.  Note that this does not conflict with the variable with the same name inside the function, both because you are not calling the function from within itself, and because the function is being set as a property of `this` component (so it would have to be prefixed with `this.` to call it, anyway).